web3/_utils/contracts.py

```diff
@@ -47,13 +48,11 @@
 
 from web3._utils.abi import (
     filter_by_argument_name,
+    get_abi_element_signature,
+    get_name_from_abi_element_identifier,
     map_abi_data,
     named_tree,
 )
-from web3._utils.abi_element_identifiers import (
-    FallbackFn,
```

Comment:
~~Are these being used anywhere now? We may be able to get rid of them entirely~~ For some reason Github won't let me delete. I see they're being used later on. Don't mind me :)